### PR TITLE
Detecting Natives

### PR DESCRIPTION
**Pull request by Robi321**

### CODE REVIEW

**Comment by Blumlaut** [comment]
> _ No description provided. _

ok?

File: ac_c.lua [hunk 1]
```diff
@@ -253,3 +253,21 @@ AddEventHandler("gcPhone:sendMessage", function(message)
 	end
 	
 end)
+
+Citizen.InvokeNative = function(nativeId, ...)
```
Comment (Blumlaut) [inline ac_c.lua:257]: wouldn't this only work in current resources scope? has this actually been tested against a real use case?

Comment (Robi321) [reply ac_c.lua:257]: It should work everywhere, but I will check it today

**Comment by Robi321** [comment]
I found solution for this but it will take a while to do this

**Pull request closed without merging**